pkg/curatedpackages/packageclient.go

```diff
@@ -46,3 +59,74 @@ func convertBundleVersionToPackageVersion(bundleVersions []packagesv1.SourceVers
 	}
 	return versions
 }
+
+func (p *PackageClient) GeneratePackages() ([]packagesv1.Package, error) {
+	packageNameToPackage := p.getPackageNameToPackage()
```

Comment:
Is this just a `packageMap`? I feel like this naming makes it confusing, even though you are literally describing the map's key/value.